### PR TITLE
logs: Add progress indicator when waiting for logs

### PR DESCRIPTION
**Pull request by vkareh**
To avoid users being concerned about their cluster not starting install,
we add a spinner while we poll the installation logs endpoint. Once the
logs are available, we remove the spinner and show the logs instead.

https://asciinema.org/a/WoaLz9coGMc5quqKjTBDRPBJY

### CODE REVIEW

**Comment by jeremyeder** [comment]
Some feedback on the video:

Drop the "Logs for cluster ... are not available yet"

Replace "it usually takes up to 5 minutes" with "$product installation beginning.  installation logs will show up within 5 minutes"

Drop the word "Waiting" and just put the dots

Drop the "cluster installation is taking longer than usual".  just continue emitting dots

**Comment by vkareh** [comment]
@jeremyeder thanks for the feedback. Done making the changes now and recorded a new video: https://asciinema.org/a/0jSlv8z7uDIkyRWav0qeEOwEl

File: cmd/logs/install/cmd.go [hunk 1]
```diff
@@ -212,6 +209,8 @@ func printLog(logs *cmv1.Log) {
 	lines := findNextLines(logs)
 	if lines != "" {
 		fmt.Printf("%s\n", lines)
+	} else {
+		fmt.Printf(".")
```
Comment (jharrington22) [inline cmd/logs/install/cmd.go:213]: I was thinking instead we could use something like .9 [here](https://godoc.org/github.com/briandowns/spinner) so we'd just have a spinner `... show up within 5 minutes  <spinner>`

Thought @vkareh @jeremyeder ?

Comment (vkareh) [reply cmd/logs/install/cmd.go:213]: I like it - will play around with it and push a change

**Comment by vkareh** [comment]
@jharrington22 @jeremyeder - I've added the spinner, and here's the new recording: https://asciinema.org/a/WoaLz9coGMc5quqKjTBDRPBJY

**Comment by jharrington22** [comment]
Nice @vkareh

**Comment by jeremyeder** [comment]
lgtm

**Pull request merged**